include/OB_table/assert.h

```diff
@@ -1,7 +1,7 @@
 #ifndef VV_ASSERT_INCLUDED
 #define VV_ASSERT_INCLUDED
 
-#include "util.h"
+#include "OB_table/util.h"
```

Comment:
Hi, lines like this should either change to `<OB_table/util.h>` or stay the same, because `""` is for local lookup and `<>` is for using the -I and system include paths.

Reply:
@Veltas : changed now and "make tests" passes. Thanks!